composer.json

```diff
@@ -12,6 +12,9 @@
     "require": {
         "php": ">=5.3.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^4"
```

Comment:
That should be removed, I never require phpunit in my composer.json. We can use symfony/phpunit-bridge instead.

Reply:
Hi.

I don't know what what should be here if you want to run phpunit in travis.
Having also dropped support for PHP < 7, there's a minimum version of PHPUnit of 6 (it seems).

If you can "fix" this so that the tests are run in travis, then I'd be happy with that.